libs/hdf-converters/test/mappers/forward/msft_secure_score_mapper.spec.ts

```diff
@@ -0,0 +1,48 @@
+import fs from 'fs';
+import {MsftConfigMapper} from '../../../src/msft-config-mapper';
+import {omitVersions} from '../../utils';
+
+describe('msft_secure_score_mapper', () => {
+  it('Successfully converts Microsoft Secure Score reports', () => {
+    const mapper = new MsftConfigMapper(
+      JSON.stringify({
+        secureScore: JSON.parse(
+          fs.readFileSync(
+            'sample_jsons/msft_secure_score_mapper/secureScore.json',
+            {
+              encoding: 'utf-8'
+            }
+          )
+        ),
+        profiles: JSON.parse(
+          fs.readFileSync(
+            'sample_jsons/msft_secure_score_mapper/profiles.json',
+            {
+              encoding: 'utf-8'
+            }
+          )
+        )
+      })
+    );
+
+    /* tslint:disable */
```

Comment:
Remove this TSLint flag. I can handle the SonarCloud flag for this code if that's what prompted you to do this.